test/uk/gov/hmrc/cataloguefrontend/FakeApplicationBuilder.scala

```diff
@@ -61,4 +72,18 @@ trait FakeApplicationBuilder
         ))
       .build()
 
+  def cookieForAuth(value: String): WSCookie = {
+    val sessionCookie =
+      sessionCookieBaker
+        .encodeAsCookie(Session(Map(SessionKeys.authToken -> value)))
+    DefaultWSCookie(
+      sessionCookie.name,
+      sessionCookieCrypto.crypto.encrypt(PlainText(sessionCookie.value)).value
+    )
+  }
+
+  def authed(path: String, token: String) =
```

Comment:
Should define the return type for public functions.
Can we replace this with `def withAuthToken(request: WSRequest, token: String): WSRequest`? This would be more flexible, and keep the `wsClient` usage with the rest (in the test)